tutor/commands/dev.py

```diff
@@ -51,7 +52,10 @@ def _stop_on_local_start(root: str, config: Config, project_name: str) -> None:
     started.
     """
     runner = DevTaskRunner(root, config)
-    if project_name != runner.project_name:
+    if (
+        runner.project_name in str(check_output("docker", "compose", "ls"))
```

Comment:
We can have a more fine-grained criterion here. Something like:

```
names = [entry["Name"] for entry in json.loads(str(check_output("docker", "compose", "ls", "--format=json"))))
if runner_project_name in names: ...
```

And the code should be de-duplicated in a shared function.